quarkus-debezium-parent/quarkus-debezium-engine-parent/deployment/src/main/java/io/quarkus/debezium/deployment/engine/InvokerGenerator.java

```diff
@@ -23,14 +26,25 @@
 import io.quarkus.gizmo.ResultHandle;
 import io.quarkus.runtime.util.HashUtil;
 
-public class InvokerGenerator {
+/**
+ *
+ * this generator is substituted by {@link CapturingInvokerGenerator} which use {@link io.debezium.runtime.CapturingEvent}
+ *
+ */
+@Deprecated
+public class InvokerGenerator implements CapturingInvokerGenerator {
```

Comment:
I don't think we need to worry about using `@Deprecated` at this stage. I suggest removing the implementation altogether.

Reply:
ok!